### PR TITLE
bin/test-lxd-network: Stop then delete instances

### PR DESCRIPTION
**Pull request by tomponline**
Part of trying to identify the cause of intermittent deletion errors

Signed-off-by: Thomas Parrott <thomas.parrott@canonical.com>

### CODE REVIEW

File: bin/test-lxd-network [hunk 1]
```diff
@@ -186,10 +186,15 @@ lxc list
...
-lxc delete -f c1-sriov
+echo "=> Removing some instances"
+lxc stop -f v1-macvlan
+lxc delete v1-macvlan
+lxc stop -f c1-macvlan
+lxc delete c1-macvlan
+lxc stop -f v1-sriov
+lxc delete v1-sriov
+lxc stop -f c1-sriov
+lxc delete c1-sriov
```
Comment (stgraber) [inline bin/test-lxd-network:197]: If it's the same thing I've seen last time, then it won't help.

Last I was on the system and hit the issue, the problem was that `v1-sriov` or some other instance had file descriptors to files from `c1-macvlan` which then prevented the deletion of `c1-macvlan` until `v1-sriov` was deleted.

Comment (tomponline) [reply bin/test-lxd-network:197]: Interesting, when/how did you ascertain that? I've tried doing that test in a loop repeatedly and wasn't able to recreate.

Comment (stgraber) [reply bin/test-lxd-network:197]: I guess I got lucky, I just deployed lantea, transfered maas-kernel, ran `maas-kernel iommu`, then transfered `lxd-test-network` and ran that, hit the issue on first go.

Comment (tomponline) [reply bin/test-lxd-network:197]: Did you see the file descriptors hanging around or just the mount error?

I'll try deploying lantea and see if I can reproduce there. What is the maas-kernel iommu thing you mentioned there?

**Pull request closed without merging**